bin/MadGraph5_aMCatNLO/cards/pp_H_TTG_012j_5f_NLO_MidDM/pp_H_TTG_012j_5f_NLO_MidDM_param_card.dat

```diff
@@ -0,0 +1,90 @@
+##        present in the decay.py files of the model.               ##
+##        See  arXiv:1402.1178 for more details.                    ##
+##                                                                  ##
+######################################################################
+
+###################################
+## INFORMATION FOR CKMBLOCK
+###################################
+Block ckmblock 
+    1 2.277360e-01 # cabi 
+
+###################################
+## INFORMATION FOR MASS
+###################################
+Block mass 
+    1 5.040000e-03 # MD 
+    2 2.550000e-03 # MU 
+    3 1.010000e-01 # MS 
+    4 1.270000e+00 # MC 
+    5 4.700000e+00 # MB 
+    6 1.720000e+02 # MT 
+   11 5.110000e-04 # Me 
+   13 1.056600e-01 # MMU 
+   15 1.777000e+00 # MTA 
+   23 9.118760e+01 # MZ 
+   25 1.250000e+02 # MH 
+## Dependent parameters, given by model restrictions.
+## Those values should be edited following the 
+## analytical expression. MG5 ignores those values 
+## but they are important for interfacing the output of MG5
+## to external program such as Pythia.
+  12 0.000000e+00 # ve : 0.0 
+  14 0.000000e+00 # vm : 0.0 
+  16 0.000000e+00 # vt : 0.0 
+  21 0.000000e+00 # g : 0.0 
+  22 0.000000e+00 # a : 0.0 
+  24 7.982436e+01 # w+ : cmath.sqrt(MZ__exp__2/2. + cmath.sqrt(MZ__exp__4/4. - (aEW*cmath.pi*MZ__exp__2)/(Gf*sqrt__2))) 
+
+###################################
+## INFORMATION FOR SMINPUTS
+###################################
+Block sminputs 
+    1 1.279000e+02 # aEWM1 
+    2 1.166370e-05 # Gf 
+    3 1.184000e-01 # aS (Note that Parameter not used if you use a PDF set) 
+
+###################################
+## INFORMATION FOR YUKAWA
+###################################
+Block yukawa 
+    1 5.040000e-03 # ymdo 
+    2 2.550000e-03 # ymup 
+    3 1.010000e-01 # yms 
+    4 1.270000e+00 # ymc 
+    5 4.700000e+00 # ymb 
+    6 1.720000e+02 # ymt 
+   11 5.110000e-04 # yme 
+   13 1.056600e-01 # ymm 
+   15 1.777000e+00 # ymtau 
+
+###################################
+## INFORMATION FOR DECAY
+###################################
+DECAY   6 1.508336e+00 # WT 
+DECAY  23 2.495200e+00 # WZ 
+DECAY  24 2.085000e+00 # WW 
+DECAY  25 4.070000e-03 # WH 
+## Dependent parameters, given by model restrictions.
+## Those values should be edited following the 
+## analytical expression. MG5 ignores those values 
+## but they are important for interfacing the output of MG5
+## to external program such as Pythia.
+DECAY  1 0.000000e+00 # d : 0.0 
+DECAY  2 0.000000e+00 # u : 0.0 
+DECAY  3 0.000000e+00 # s : 0.0 
+DECAY  4 0.000000e+00 # c : 0.0 
+DECAY  5 0.000000e+00 # b : 0.0 
+DECAY  11 0.000000e+00 # e- : 0.0 
+DECAY  12 0.000000e+00 # ve : 0.0 
+DECAY  13 0.000000e+00 # mu- : 0.0 
+DECAY  14 0.000000e+00 # vm : 0.0 
+DECAY  15 0.000000e+00 # ta- : 0.0 
+DECAY  16 0.000000e+00 # vt : 0.0 
+DECAY  21 0.000000e+00 # g : 0.0 
+DECAY  22 0.000000e+00 # a : 0.0 
```

Comment:
instead of using parameter card, please use customized card, in which only the modified values are included comparing with the default parameter card in the model.